config/import-control.xml

```diff
@@ -60,7 +59,6 @@
   </file>
 
   <subpackage name="utils">
-    <!-- until https://github.com/checkstyle/checkstyle/issues/6628 -->
     <allow pkg="antlr"/>
     <allow class="com.puppycrawl.tools.checkstyle.DetailAstImpl"/>
```

Comment:
@rnveach same here

Reply:
@romani Same here, and restrict `antlr` even further?

Reply:
https://github.com/checkstyle/checkstyle/issues/9272 - new reason, I think fix will be easy

Reply:
@strkkk , please use #9272

Reply:
done